### PR TITLE
add in jenkins files

### PR DESCRIPTION
**Pull request by rybit**

### CODE REVIEW

File: script/ci.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+set -e
+set -x
+
+WORKSPACE=/go/src/github.com/netlify/netlify-commons
+
+docker run \
+	--volume $(pwd):$WORKSPACE \
+	--workdir $WORKSPACE \
+	--rm \
+	calavera/go-glide:v0.12.3 script/test.sh
```
Comment (brycekahle) [inline script/ci.sh]: This can be `netlify/go-glide:v0.12.3` now.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+PROJECT = service-boilerplate
+
+help: ## Show this help.
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {sub("\\\\n",sprintf("\n%22c"," "), $$2);printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+all: deps test build ## Run the tests and build the binary.
+
+deps: ## Install dependencies.
+	go get -u github.com/Masterminds/glide && glide --home /tmp install
```
Comment (brycekahle) [inline Makefile]: Why `/tmp`?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.PONY: all build deps image test
+
+PROJECT = service-boilerplate
```
Comment (brycekahle) [inline Makefile]: Change to `netlify-commons`?

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,14 @@
+.PONY: all build deps image test
```
Comment (sgirones) [inline Makefile]: s/PONY/PHONY/

File: Makefile (hunk 2 above)
Comment (rybit) [inline Makefile]: otherwise it can pollute the working directory because it is mounted

File: Makefile (hunk 3 above)
Comment (rybit) [inline Makefile]: copy pasta error

File: Makefile (hunk 4 above)
Comment (rybit) [inline Makefile]: HA! this is in all our makefiles...

**Pull request merged**